src/models/dynamodb.ts

```diff
@@ -0,0 +1,39 @@
+import AWS from "aws-sdk";
```

Comment:
Curious if it needs to be in models? I think models are only about sequelize. Should we maybe create `dynamodb` dir and pull all dynamodb-related stuff there?

Reply:
Well, I am against put it under a dir name `dynamodb` as I think dir should be divided by 'features' instead of 'technologies`.

But putting under `models` might not make sense either. This is more like a services utils. And from the name of the methods, it doesn't even imply "dynamo" as the technologies.

I'll put it under the `services` dir?

Reply:
done.

Reply:
Sorry, I might miss something but the file is still in models dir, isn't it? :badpokerface:

Reply:
Sorry I missed this one. I moved the deployment-service to under /services. 
For this one, I split it into two now. The `db connection` moved to "config", and the `purgeItem` into the test/setup/setup.ts coz it is only used by test and only in that location.

Now should looks better.